omero/sysadmins/unix/server-install-homebrew.rst

```diff
@@ -41,175 +52,262 @@ running::
+
+    export PATH="/usr/local/opt/python/libexec/bin:$PATH"
+
+5. Check that Python is working and is version 2.7.x::
 
     $ which python
-    /usr/local/bin/python
+    /usr/local/opt/python/libexec/bin/python
+
     $ python --version
-    Python 2.7.9
+    Python 2.7.13
 
-The installation of OMERO via Homebrew depends upon two alternate
-repositories containing extra formulae:
-https://github.com/Homebrew/homebrew-science for the HDF5 formula and
-https://github.com/ome/homebrew-alt for all the OME-provided formulae
-and older versions of Ice. To add these, run::
+6. For developing with OMERO, or Python in general, we recommend the use of Virtualenv.
+   Virtualenv allows development of Python applications without having to
+   worry about clashing third-party packages for different Python projects.
 
-    $ brew tap homebrew/science
-    $ brew tap ome/alt
+   Use pip to get `Virtualenv <https://virtualenv.pypa.io/en/stable/>`__::
 
-.. note::
+    $ pip install virtualenv
+
+   With Virtualenv installed, create a virtual environment::
+
+    $ virtualenv ~/Virtual/omero
+
+   This will create a folder to hold Python libraries in the the directory :file:`~/Virtual/omero/lib`
+
+  .. note:: 
+   You can activate the Virtualenv environment that we created using::
+
+    $ source ~/Virtual/omero/bin/activate
+
+   This will swtich to using Pip and Python in the Virtualenv directory 
+   :file:`~/Virtual/omero/bin` and any Pip libraries you install, whilst the Virtualenv is activated, 
+   will be installed to :file:`source ~/Virtual/omero/lib`.
+
+  .. note::
+   **(Optional)** To make starting a Virtualenv environment easier,
+   you can add an `alias` to your :file:`.bash_profile`::
+
+    alias startVmOmero="source ~/Virtual/omero/bin/activate"
 
-    The Homebrew formulae used below provide Python bindings. As
-    described in `Homebrew and Python`_, you should **not** be in an
-    active virtual environment when you ``brew install`` them.
+   Using the command-line terminal, reload your :file:`.bash_profile`::
 
-See the :download:`step01_deps.sh <walkthrough/osx/step01_deps.sh>` script for
-the steps described above.
+    $ source ~/.bash_profile
+
+   Now you can activate the Virtualenv environment using::
+
+    $ startVmOmero
 
 OMERO installation
 ------------------
 
-OMERO |release| server
-^^^^^^^^^^^^^^^^^^^^^^
+Pre-built server
+^^^^^^^^^^^^^^^^
+
+1. Using the command-line terminal, prepare a place for your OMERO server to 
+   be downloaded to.
+
+   Find the current OMERO.server zip from the
+   `downloads page <https://downloads.openmicroscopy.org/latest/omero/artifacts/>`_.
+   Download and extract the OMERO.server-x.x.x-ice36-bxx.zip.
+
+2. Once extracted, open your :file:`.bash_profile` in a text editor, 
+   such as the built-in TextEdit app::
+
+    $ open -a TextEdit.app ~/.bash_profile
+
+   Add an environment variable :envvar:OMERO_SERVER to the :file:`.bash_profile` which points
```

Comment:
need the backticks

Reply:
ditto, the :envvar:OMERO_SERVER does not build correctly